helper/pq/pq_test.go

```diff
@@ -16,17 +16,17 @@ func TestGetTableQuery(t *testing.T) {
 	}
 }
 
-func TestTruncateTableCommand(t *testing.T) {
+func TestTruncateTablesCommand(t *testing.T) {
 	helper := postgres.Helper{}
-	cmd := helper.TruncateTableCommand("users")
+	excludedTables := []string{"users"}
```

Comment:
in case we have `if len(tableNames)` as above, there should be a testcase to cover that.